### DISCUSSION TITLE
My-CS-Notebook/about/

### DISCUSSION THREAD

**Discussion by giscus** [Zhang-Each/My-CS-Notebook#4] [General]
# My-CS-Notebook/about/

https://zhang-each.github.io/My-CS-Notebook/about/

**Comment by IsshikiHugh** [comment]
膜小角龙前辈！

**Reply by Zhang-Each** [reply]
欢迎多来玩呀～

**Comment by MiraclePlusWill** [comment]
前辈可以给个邮箱或者什么的联系方式嘛，想请教您一些问题

**Comment by Cstardust** [comment]
强啊大佬，刷了这么多网课👍来向大佬学习，希望以后有机会换下友链哈哈哈

**Comment by MiraclePlusWill** [comment]
您好，我已收到您的邮件，将尽快给您回复。

**Comment by Monyan** [comment]
大佬好厉害，可以给个联系方式吗？55555

**Comment by MiraclePlusWill** [comment]
您好，我已收到您的邮件，将尽快给您回复。

**Comment by xuwenlong1** [comment]
前辈你好，我是即将入学的zju硕，也想学习一下nlp kg llm相关的，前辈可以推荐一些入门教程或者路线么

**Comment by MiraclePlusWill** [comment]
您好，我已收到您的邮件，将尽快给您回复。

**Comment by xufangfang408** [comment]
太强了吧

**Comment by MiraclePlusWill** [comment]
您好，我已收到您的邮件，将尽快给您回复。
